pkg/apis/admissionregistration/types.go

```diff
@@ -223,7 +223,7 @@ type Webhook struct {
 	// +optional
 	NamespaceSelector *metav1.LabelSelector
 
-	// SideEffects states whether this webhookk has side effects.
```

Comment:
Can you fix the typo in this [file](https://github.com/kubernetes/kubernetes/blob/8ae998ceb69ae83afe730795aea3bd44913ad868/staging/src/k8s.io/api/admissionregistration/v1beta1/types.go#L235) and run hack/update-all.sh to update the generated public docs?

Reply:
Thanks for the reminder. Done